model/src/main/java/org/cbioportal/model/SampleClinicalDataCollection.java

```diff
@@ -0,0 +1,19 @@
+package org.cbioportal.model;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SampleClinicalDataCollection {
```

Comment:
I do not fully understand this, how about patient-level data? I can see that we have `patientClinicalData` in `ClinicalDataCollection` before.

Reply:
Yes indeed. The table shown in the clinical tab on study view is a _sample_ centric table (every sample is a row). The patient attributes are added to each line for each sample (so redundant patient data is added for samples). Before the data was sent as separate sample-level and patient-level clinical attributes to the frontend; the fontend replicated the patient data to each row. This functionality has been moved to the backend. As such there is no distinction anymore between sample and patient data in the response object.

Reply:
Got it, then I think this is fine